rest-api/test/unit_test/api_test/participant_summary_api_test.py

```diff
@@ -249,6 +250,46 @@ def testQuery_noSummaries(self):
     response = self.send_get('ParticipantSummary')
     self.assertBundle([], response)
 
+  def test_get_summary_list_returns_total(self):
+    num_participants = 200
```

Comment:
This is going to run for a while I think. I'd make num_participants much smaller and then pass up _count=<something smaller than that> to make it go faster.